supabase/pending/directives.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# This script migrates a pre-directives local stack (or production) to a post-directives install.
+# You must first create an 'accounts@estuary.dev' user or the migration will fail to apply & rollback.
+# Then rebuild / restart your control-plane agent to have it automatically provision the 'ops/' tenant.
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+function psql_input() {
+    cat <<EOF
+begin;
+
+drop table public.flow_materializations_v2 cascade;
+drop table public.task_stats_by_minute cascade;
+drop table public.flow_checkpoints_v1 cascade;
+drop table public.task_stats_by_hour cascade;
+drop table public.task_stats_by_day cascade;
```

Comment:
Don't these table not exist on prod yet? So don't we need `drop table if exists` here, otherwise this will fail?

Reply:
They do exist in prod, and we don't want them to.